### DISCUSSION TITLE
Website improvements

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#92] [A Cosmology Group]
A few suggestions

- [x] Create a new high-res ACG logo (see #106)
- [x] Update repo logo, site logo, and site favicon https://just-the-docs.com/docs/configuration/#site-logo
- [x] Change purple theme color to another color? https://just-the-docs.com/docs/utilities/color/
- [x] Change "Template Repository" on navbar to "GitHub Repo"
- [ ] Shrink font of "A Cosmology Group" in top left so it fits on one line

**Comment by ftessier** [comment]
I can work on the logo.

**Comment by ftessier** [comment]
For the color, perhaps it could be one that corresponds to a significant wavelength in cosmology, if anyone has a suggestion.

**Reply by RedshiftDrift** [reply]
Some suggestions:
- Lyman- $\alpha$ at $\lambda = 121.6$ nm,
- Balmer break at $\lambda = 364.5$ nm,
- Fraunhofer H-K lines at $\lambda \simeq 395$ nm,
- CMB at $\lambda = 1.9$ mm,
- hydrogen hyperfine structure at $\lambda = 21$ cm.

**Reply by ftessier** [reply]
According to https://www.johndcook.com/wavelength_to_RGB.html, 395 nm corresponds to colour `#8000a1`, which is indeed purple.

**Reply by budrap00** [reply]
I'm down with the 395 purple!

**Reply by RedshiftDrift** [reply]
Edwin Hubble's success depended on the purple [H-K lines](https://www.wwu.edu/astro101/a101_hubble_redshift.shtml)

**Reply by ftessier** [reply]
I tried to implement this in #94, but I have no idea if it is going to work, I don't know how to test this offline...

**Reply by RedshiftDrift** [reply]
Some tips when testing webpage deployment:
- make sure to clear your browser's cache, the new website settings may not be updated in your browser,
- look for error messages in [`Deploy Jekyll site to Pages`](https://github.com/a-cosmology-group/acg/actions/runs/5469181933), the green ✔️ shows there is no error on the last deploy,
- search [Discussions on just-the-docs](https://github.com/just-the-docs/just-the-docs/discussions) for similar problems (they helped me with the `Pages` setup!)

**Reply by ftessier** [reply]
Thanks! I am installing Jekyll locally (using `rbenv` to sidestep the outdated system `ruby`) for testing. Almost there...

**Comment by RedshiftDrift** [comment]
Most of these are configurable by editing the `_config.yml` file [https://github.com/a-cosmology-group/acg/blob/main/_config.yml](https://github.com/a-cosmology-group/acg/blob/main/_config.yml)

The parameters are described here [https://just-the-docs.com/docs/configuration/](https://just-the-docs.com/docs/configuration/)

@sahil5d you are encouraged to try it!

**Comment by RedshiftDrift** [comment]
@ftessier I uploaded the new logo to the `Profile picture` but the image needs to be square, see
 [https://github.com/organizations/a-cosmology-group/settings/profile](https://github.com/organizations/a-cosmology-group/settings/profile).

Perhaps extra top and bottom borders would work well?

**Reply by ftessier** [reply]
Ok, I'll update the images, increasing the vertical size from 231 pixels back to 256.

**Reply by ftessier** [reply]
Done, I pushed to `main` directly.

**Reply by ftessier** [reply]
I update the organization image. Looks like it takes a little while to update the actual icon?

**Reply by ftessier** [reply]
All good, it works, of course I had to `shift`-reload!

**Reply by RedshiftDrift** [reply]
Do you see it as the logo on the web page?  I still don't...

**Reply by ftessier** [reply]
I do, reload while holding down the shift key, to bypass the cache. Let me know!
